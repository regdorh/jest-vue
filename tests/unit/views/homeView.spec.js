import { shallowMount } from '@vue/test-utils'
import HomeView from '@/views/HomeView.vue'

describe('HomeView.vue', () => {
    test('ValidaciÃ³n de match con el snapshot', () => {
        const wrapper = shallowMount(HomeView)

        expect( wrapper.html() ).toMatchSnapshot()
    })
})