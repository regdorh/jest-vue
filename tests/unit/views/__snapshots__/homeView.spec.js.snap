exports[`HomeView.vue Validaci√≥n de match con el snapshot 1`] = `
<div class="home"><img alt="Vue logo" src="">
  <posts-view-stub></posts-view-stub>
</div>
`;